stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -573,8 +580,9 @@ internal class ChannelControllerImpl(
         }
 
         newMessage.user = domainImpl.currentUser
+
         // TODO: type should be a sealed/class or enum at the client level
-        newMessage.type = if (newMessage.text.startsWith("/")) {
+        newMessage.type = if (newMessage.text.startsWith("/") || hasAttachments) {
```

Comment:
Nit: This could be extracted to some `fun getMessageType(message: Message): String`